indico/modules/events/static/templates/emails/paper_judgment_notification_email.txt

```diff
@@ -0,0 +1,39 @@
+{% extends 'emails/base.txt' %}
+
+{% macro get_comments() %}
+    {% filter dedent -%}
+        Please see the comments below from the reviewing team:
```

Comment:
Either
_"Please see **below** the comments from the reviewing team"_ or 
_"Please see the comments from the reviewing team **below**"_